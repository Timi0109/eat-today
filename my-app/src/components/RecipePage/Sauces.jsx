import React from "react";
import styles from "./RecipePage.module.css";

const Sauces = () => {
    return(
        <div className={styles.saucesContainer}>
            <label className={styles.saucesLabel}>
                Soy Sauce
                <input
                className={styles.sauceInput}
                type="checkbox"
                />
            </label>
            <label className={styles.saucesLabel}>
                Sweet Sauce
                <input
                className={styles.sauceInput}
                type="checkbox"
                />
            </label>
            <label className={styles.saucesLabel}>
                Spicy Sauce
                <input
                className={styles.sauceInput}
                type="checkbox"
                />
            </label>
            <label className={styles.saucesLabel}>
                Salad Sauce
                <input
                className={styles.sauceInput}
                type="checkbox"
                />
            </label>
            <label className={styles.saucesLabel}>
                Salt
                <input
                className={styles.sauceInput}
                type="checkbox"
                />
            </label>
            <label className={styles.saucesLabel}>
                Oil
                <input
                className={styles.sauceInput}
                type="checkbox"
                />
            </label>
            <label className={styles.saucesLabel}>
                Butter
                <input
                className={styles.sauceInput}
                type="checkbox"
                />
            </label>
            
        </div>
    )
}

export default Sauces;