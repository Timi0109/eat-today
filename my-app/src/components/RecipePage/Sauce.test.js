import React from "react";
import Sauces from "./Sauces";

describe("Sauces tests", () => {
  it("renders correctly", () => {
    const { container } = render(<Sauces></Sauces>);
    expect(container).toMatchSnapshot();
  });

});
