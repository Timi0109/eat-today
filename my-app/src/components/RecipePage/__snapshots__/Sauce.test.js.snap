// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sauces tests renders correctly 1`] = `
<div>
  <div
    class="saucesContainer"
  >
    <label
      class="saucesLabel"
    >
      Soy Sauce
      <input
        class="sauceInput"
        type="checkbox"
      />
    </label>
    <label
      class="saucesLabel"
    >
      Sweet Sauce
      <input
        class="sauceInput"
        type="checkbox"
      />
    </label>
    <label
      class="saucesLabel"
    >
      Spicy Sauce
      <input
        class="sauceInput"
        type="checkbox"
      />
    </label>
    <label
      class="saucesLabel"
    >
      Salad Sauce
      <input
        class="sauceInput"
        type="checkbox"
      />
    </label>
    <label
      class="saucesLabel"
    >
      Salt
      <input
        class="sauceInput"
        type="checkbox"
      />
    </label>
    <label
      class="saucesLabel"
    >
      Oil
      <input
        class="sauceInput"
        type="checkbox"
      />
    </label>
    <label
      class="saucesLabel"
    >
      Butter
      <input
        class="sauceInput"
        type="checkbox"
      />
    </label>
  </div>
</div>
`;
